packages/contracts/contracts/optimistic-ethereum/OVM/verification/OVM_BondManager.sol

```diff
@@ -1,211 +1,91 @@
-        // slash!
-        bond.state = State.NOT_COLLATERALIZED;
-    }
-
-    /// Sequencers call this function to post collateral which will be used for
-    /// the `appendBatch` call
-    function deposit() override public {
-        require(
-            token.transferFrom(msg.sender, address(this), requiredCollateral),
-            Errors.ERC20_ERR
-        );
-
-        // This cannot overflow
-        bonds[msg.sender].state = State.COLLATERALIZED;
-    }
-
-    /// Starts the withdrawal for a publisher
-    function startWithdrawal() override public {
-        Bond storage bond = bonds[msg.sender];
-        require(bond.withdrawalTimestamp == 0, Errors.WITHDRAWAL_PENDING);
-        require(bond.state == State.COLLATERALIZED, Errors.WRONG_STATE);
-
-        bond.state = State.WITHDRAWING;
-        bond.withdrawalTimestamp = uint32(block.timestamp);
-    }
-
-    /// Finalizes a pending withdrawal from a publisher
-    function finalizeWithdrawal() override public {
-        Bond storage bond = bonds[msg.sender];
-
-        require(
-            block.timestamp >= uint256(bond.withdrawalTimestamp) + disputePeriodSeconds,
-            Errors.TOO_EARLY
-        );
-        require(bond.state == State.WITHDRAWING, Errors.SLASHED);
-
-        // refunds!
-        bond.state = State.NOT_COLLATERALIZED;
-        bond.withdrawalTimestamp = 0;
-
-        require(
-            token.transfer(msg.sender, requiredCollateral),
-            Errors.ERC20_ERR
-        );
-    }
-
-    /// Claims the user's reward for the witnesses they provided for the earliest
-    /// disputed state root of the designated publisher
-    function claim(address who) override public {
-        Bond storage bond = bonds[who];
-        require(
-            block.timestamp >= bond.firstDisputeAt + multiFraudProofPeriod,
-            Errors.WAIT_FOR_DISPUTES
-        );
-
-        // reward the earliest state root for this publisher
-        bytes32 _preStateRoot = bond.earliestDisputedStateRoot;
-        Rewards storage rewards = witnessProviders[_preStateRoot];
-
-        // only allow claiming if fraud was proven in `finalize`
-        require(rewards.canClaim, Errors.CANNOT_CLAIM);
-
-        // proportional allocation - only reward 50% (rest gets locked in the
-        // contract forever
-        uint256 amount = (requiredCollateral * rewards.gasSpent[msg.sender]) / (2 * rewards.total);
-
-        // reset the user's spent gas so they cannot double claim
-        rewards.gasSpent[msg.sender] = 0;
-
-        // transfer
-        require(token.transfer(msg.sender, amount), Errors.ERC20_ERR);
-    }
+    function recordGasSpent(
+        bytes32 _preStateRoot,
+        bytes32 _txHash,
+        address _who,
+        uint256 _gasSpent
+    )
+        override
+        public
+    {}
+
+    function finalize(
+        bytes32 _preStateRoot,
+        address _publisher,
+        uint256 _timestamp
+    )
+        override
+        public
+    {}
+
+    function deposit()
```

Comment:
Must this all be deleted?

Reply:
The original code wasn't being used in production. I made a comment in #protocol about this. I felt like having a "mock" contract in production was confusing. Seemed better to replace the code with the mockOVM_BondManager that was actually being used (at least for now)